xbstrap/__init__.py

```diff
@@ -918,6 +941,28 @@ def resolve_host_paths(x):
 # ----------------------------------------------------------------------------------------
 
 
+def do_maintainer(args):
+    cfg = config_for_args(args)
+    xmu_dir = os.path.join(_util.find_home(), "bin", "xmu")
+    if not os.access(os.path.join(xmu_dir, "xmu.js"), os.F_OK):
+        _util.log_err("The maintainer utilities are not installed.")
+        _util.log_info("Try running `xbstrap prereqs xmu` to install them.")
+        return
+    proc = subprocess.Popen(
+        [os.path.join(xmu_dir, "xmu.js")] + args.args, cwd=os.path.abspath(cfg.source_root)
+    )
```

Comment:
Also, we simply pass through the command here. I would prefer to use the node.js tool for YAML rewriting, but not for all possible maintainer tasks.